include/sys/vdev.h

```diff
@@ -114,7 +114,8 @@ extern void vdev_xlate_walk(vdev_t *vd, const range_seg64_t *logical_rs,
     vdev_xlate_func_t *func, void *arg);
 
 extern void vdev_get_stats_ex(vdev_t *vd, vdev_stat_t *vs, vdev_stat_ex_t *vsx);
-extern uint64_t getMirrorSlogUsedSize(vdev_t* vd);
+extern void getMirrorSlogUsedSize(vdev_t *vd, unsigned __int64
```

Comment:
can change getMirrorSlogUsedSize function name to getMirrorSlogSizeDetails

Reply:
Resolved!